src/codecoverage/backend/codecoverage_backend/__init__.py

```diff
@@ -31,4 +35,10 @@ def create_app(config=None):
     # Setup datadog stats
     codecoverage_backend.datadog.get_stats()
 
+    # Warm up GCP cache
```

Comment:
By default this is going to download the latest 5 reports, but most likely people will only look at the latest one, so maybe we should pre-download just the latest one.